source/MaterialXCore/Document.cpp

```diff
@@ -170,12 +170,21 @@ void Document::initialize()
     setVersionIntegers(MATERIALX_MAJOR_VERSION, MATERIALX_MINOR_VERSION);
 }
 
-NodeDefPtr Document::addNodeDefFromGraph(const NodeGraphPtr nodeGraph, const string& nodeDefName, const string& node,
-                                         const string& version, bool isDefaultVersion, const string& group, string& newGraphName)
+NodeDefPtr Document::addNodeDefFromGraph(const NodeGraphPtr nodeGraph, const string& nodeDefName, const string& category,
```

Comment:
@ashwinbhat, I'm don't this code is correct in general as it was added in for a specific Autodesk workflow.

Reply:
@kwokcb I don't follow, the change here is to add a namespace? How is this Autodesk specific workflow.